database/controller.js

```diff
@@ -0,0 +1,26 @@
+const db = require('./index');
+
+module.exports.getImage = (id) => {
+  const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  return db.query(queryString, [id]);
+};
+
+module.exports.addImage = (data) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
+module.exports.updateImage = (id) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
+module.exports.deleteImage = (id) => {
+  // const queryString = 'SELECT imageUrl FROM images WHERE id = (?)';
+  // return db.query(queryString, [id]);
+};
+
```

Comment:
I know that we're not using these endpoints right now, but should they be commented out? Not a big deal, though.

Reply:
Also, all of these queries look the same?

Reply:
Flushed out these queries, to be more complete!